<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartmentSelectForm</class>
 <widget class="QWidget" name="DepartmentSelectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DepartmentSelectForm{
	font: 24pt &quot;DejaVu Sans&quot;;  
    color: white;
	background:qlineargradient(spread:pad, x1:0.494318, y1:0, x2:0.494, y2:1, stop:0 rgba(0, 30, 65, 255), stop:1 rgba(112, 124, 165, 255));
}

QListView {
	font: 24pt &quot;DejaVu Sans&quot;;  
    color: black;
    show-decoration-selected: 1; /* make the selection span the entire width of the view */
}

QListView::item:alternate {
	font: 24pt &quot;DejaVu Sans&quot;;  
    color: black;
    background: #EEEEEE;
}

QListView::item:selected {
	font: 24pt &quot;DejaVu Sans&quot;;  
    color: black;
    border: 1px solid #222222;
}

QListView::item:selected:!active {
    font: 24pt &quot;DejaVu Sans&quot;;  
    color: black;
	background: rgb(133, 133, 133);
}

QListView::item:selected:active {
	font: 24pt &quot;DejaVu Sans&quot;;  
    color: white;
    background: rgb(0, 134, 201);
}


QMessageBox
{
    font: 28pt &quot;DejaVu Sans&quot;;  
    color: white;  
	background-color: qlineargradient(spread:pad, x1:0.488636, y1:0, x2:0.511364, y2:1, stop:0 rgba(136, 50, 7, 255), stop:1 rgba(200, 57, 29, 255));
}

QLabel
{
   background-color: rgba(255,255,255,10); 
   font: 24pt &quot;DejaVu Sans&quot;;  
   color: white;
}

QPushButton
{
	background-color: rgba(255,255,255,10); 
   font: 24pt &quot;DejaVu Sans&quot;;  
    color: white;
   border-style: outset;
   border-width: 2px;
   border-color: beige;
	padding:4px;
}

QPushButton:pressed 
{
	font: 24pt &quot;DejaVu Sans&quot;;  
    color: white;
	padding:4px;
	background-color: rgba(255,255,255,90); 
	border-style: inset;
}

QPushButton:disabled 
{
	font: 24pt &quot;DejaVu Sans&quot;;  
   padding:4px;
	color: rgb(83, 83, 83);
	background-color: rgba(171, 171, 171, 50);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,5">
     <item>
      <layout class="QVBoxLayout" name="layoutSelectDepartment">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>1. Select department</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidgetDepartment">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonPageUp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>UP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPageDown">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>DOWN</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>2. Select login name </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QListWidget" name="listEmployee">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonPageUpLogin">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>UP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPageDownLogin">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>DOWN</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="layoutPassword">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>3. Input password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linePassword"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLogin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLogout">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Log Out</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelLoginInformation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Department: none   Login: none</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
